### PR TITLE
Frontend of user-information-settings

### PR DESCRIPTION
**Pull request by Ivan-Edan**
**Description:** This is PR is all about the Frontend of the User Settings Module which is the Information Page.

**Information Page**
- Added front end for this information settings
- Initial responsiveness
- Modals was inserted

**What's not yet integrated/Enhance:**
- the responsiveness needs to improve
- some padding and margin needs to adjust
- The iframe for the pdf needs to improved
- submit button in enter password needs to disable first
- some icons needs to change accurately
- Need to add backend
- Modals redirection

**Email Update page**
- Added front end design
- Initial responsiveness
- Modal when successful update of email

**What's not yet integrated/Enhance**
- green check vector on the email address text field
- the responsiveness needs to improve
- some padding and margin need to adjust
- Need to add Backend
- Proper modal connection/redirection

Sample Output:
image

image

image

### CODE REVIEW

**Comment by Carl-Tabuso** [comment]
- This is mostly frontend so I think we should get the inputs of @biellamariscotes.
- And since this is a collaborative effort with Team 1, you may also seek review and approval of @Andrea-Villalobos

**Review by Andrea-Villalobos — COMMENTED** [review]
- On my device with zoom level of 100%, this is what it looks like:
image
The containers' length are not equal. Some paddings are a bit off. 
Looks crowded.

- Missing href links in the side navigation panel/section.

- The eye toggle for password is missing.
image
- It seems that the check gif is not centered, it looks like it is placed a little bit to the left. I am not sure though.
- I cannot also dismiss this modal using the close/X button.

image

- I am suggesting that we use the same css file named user-settings.css so we can make the containers uniform. Please see my transfer org page.  Or do you have any idea/s on how can we make it uniform without having conflicts in the css?

**Comment by Ivan-Edan** [comment]
Already change the style of the container using the template of transfer org made by @Andrea-Villalobos

**Comment by Ivan-Edan** [comment]
Already fix the redirection/linking of the pages in settings module and also renamed the file of transfer-org.php -> user-setting-transfer.php for consistency of the other parts of the setting module.

Review required: @Andrea-Villalobos

**Comment by biellamariscotes** [comment]
There seems to be a conflict with the development branch. Please address it first before I can review your changed files. Thank you! @Ivan-Edan

File: src/includes/components/topnavbar.php [hunk 1]
```diff
@@ -10,6 +10,8 @@
         <i id="dropdown-chevron" class="fas fa-chevron-down ps-1"></i>
       </a>
       <ul class="dropdown-menu dropdown-menu-end px-2 py-3" style="font-size: 12px; font-weight:500;" aria-labelledby="navbarDropdown">
+        <li><a class="dropdown-item" href="user-setting-information.php"><span class="main-color pe-3"><i data-feather="settings"></i></span>Settings</a></li>
+        <li><a class="dropdown-item" href="#"><span class="main-color pe-3"><i data-feather="help-circle"></i></span>FAQs</a></li>
         <li><a class="dropdown-item" href="#"><span class="main-color pe-3"><i data-feather="settings"></i></span>Settings</a></li>
         <li><a class="dropdown-item" href="voter-faqs.php"><span class="main-color pe-3"><i data-feather="help-circle"></i></span>FAQs</a></li>
```
Comment (biellamariscotes) [inline src/includes/components/topnavbar.php]: There's a duplication here. Please remove the extra links.

image

File: src/setting-email-update.php [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <!-- Fontawesome CDN Link -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
+    <!-- Bootstrap 5 code -->
+    <link type="text/css" href="../vendor/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="styles/loader.css" />
+    <link rel="stylesheet" href="styles/feedback-suggestions.css" />
+    <link rel="stylesheet" href="styles/setting-email.css" />
+    <link rel="stylesheet" href="<?php echo '../src/styles/orgs/' . $org_acronym . '.css'; ?>">
```
Comment (biellamariscotes) [inline src/setting-email-update.php:38]: There is an absence of main styles component here: `style.css `and `core.css`

**Review by biellamariscotes — CHANGES_REQUESTED** [review]
Please read the comments.

**Comment by biellamariscotes** [comment]
Based on my comment on lines +34 to +38, there is an issue with the footer.

Since the footer's front-end relies on the style.css and core.css, I tried applying these styles on my end. However, it is not properly rendering, and this is because of the conflicts between duplicate classes in the main style components and the file's own styles. See the outcome below:

image

Kindly tell me if you guys will apply your own style of the footer (but this should look identical to the admin side's footer) or fix the conflicting styles.

**Comment by biellamariscotes** [comment]
Will merge this now. As what we've agreed on in our conversion (f2f), the responsiveness of the footer shall be resolved on the next PR. Thanks!

**Review by biellamariscotes — APPROVED** [review]

**Pull request merged**